config/config.php

```diff
@@ -13,5 +13,14 @@
                 'version' => '1.0.0',
             ],
         ],
-    ]
+    ],
+
+    /**
+     * The storage disk to be used to place the generated `*_openapi.json` or `*_openapi.yaml` file.
+     *
+     * For example, if you use 'public' you can access the generated file as public web asset (after run `php artisan storage:link`).
+     *
+     * Supported: 'local', 'public' and (probably) any disk available in your filesystems (https://laravel.com/docs/9.x/filesystem#configuration).
+     */
+    'filesystem_disk' => env('OPEN_API_SPEC_GENERATOR_FILESYSTEM_DISK', 'local'),
```

Comment:
Currently we use the default disk and although we assume it is 'local', it doesn't necessarily have to be that disk. By forcing the disk to be 'local' we break that functionality. A default of `null` would be better as that will result in the current behaviour; the default disk. We can then explain this behaviour in de doc above.
```suggestion
    'filesystem_disk' => env('OPEN_API_SPEC_GENERATOR_FILESYSTEM_DISK', null),
```